sdks/wasm/DebuggerTestSuite/PossibleBreakpointsTest.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace DebuggerTests
+{
+	public class PossibleBreakpointsTest : DebuggerTestBase
+	{
+		const string JsTestFile = "/possible-breakpoints.js";
+		const int JsStartLine = 2;
+
+		const string ManagedTestFile = "dotnet://debugger-test.dll/possible-breakpoints.cs";
+		const int ManagedStartLine = 5;
+
+		[Fact]
+		public Task TestGetPossibleBreakpointsJsNoEnd () => TestGetPossibleBreakpoints (JsTestFile, JsStartLine, null, new ExpectedLocation[] {
+				new ExpectedLocation (JsStartLine, 1),
+				new ExpectedLocation (JsStartLine, 9, "call"),
+				new ExpectedLocation (JsStartLine + 1, 1),
+				new ExpectedLocation (JsStartLine + 1, 9, "call"),
+				new ExpectedLocation (JsStartLine + 2, 0, "return"),
+				new ExpectedLocation (JsStartLine + 5, 1),
+				new ExpectedLocation (JsStartLine + 5, 9, "call"),
+				new ExpectedLocation (JsStartLine + 6, 0, "return")
+			});
+
+		[Fact]
+		public Task TestGetPossibleBreakpointsJsWithEnd () => TestGetPossibleBreakpoints (JsTestFile, JsStartLine, JsStartLine + 5, new ExpectedLocation[] {
+				new ExpectedLocation (JsStartLine, 1),
+				new ExpectedLocation (JsStartLine, 9, "call"),
+				new ExpectedLocation (JsStartLine + 1, 1),
+				new ExpectedLocation (JsStartLine + 1, 9, "call"),
+				new ExpectedLocation (JsStartLine + 2, 0, "return")
+			});
+
+		[Fact]
+		public Task TestGetPossibleBreakpointsJsEndBeforeStart () => TestGetPossibleBreakpoints (JsTestFile, JsStartLine + 5, JsStartLine, new ExpectedLocation[0]);
+
+		[Fact]
+		public Task TestGetPossibleBreakpointsJsHighStart () => TestGetPossibleBreakpoints (JsTestFile, 9999, null, new ExpectedLocation[0]);
+
+		[Fact]
+		public Task TestGetPossibleBreakpointsJsNevativeStart () => TestGetPossibleBreakpoints (JsTestFile, -1, null, new ExpectedLocation[0]);
```

Comment:
s,Nevative,Negative,

Reply:
Oops, fixed.  Thanks for spotting this typo!